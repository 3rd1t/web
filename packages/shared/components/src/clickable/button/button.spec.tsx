import React from "react"
import { render } from "@testing-library/react"

import Button from "./button"

describe("Button", () => {
  it("should render successfully", () => {
    const { baseElement } = render(<Button label="Label" textColor="text-gray-900" bgColor="bg-gray-200" onClick={() => {}} />)
    expect(baseElement).toMatchSnapshot()
  })
})
