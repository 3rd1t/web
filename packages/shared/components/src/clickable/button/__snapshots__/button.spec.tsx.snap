// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render successfully 1`] = `
<body>
  <div>
    <button
      class="focus:outline-none  text-gray-900 hover:text-gray-800 bg-gray-200 hover:bg-gray-300"
    >
      <div
        class="flex items-center justify-center px-4 py-2 space-x-2 font-medium"
      >
        <span />
        <span>
          Label
        </span>
        <span />
      </div>
    </button>
  </div>
</body>
`;
